global.js

```diff
@@ -57,6 +58,10 @@ window.onload = function() {
   var submitButton = document.getElementById("submitter");
   submitButton.addEventListener("click", function () {
     // need to add a function that does something with the user's answer.
+    var questionResult = getElementById("question_result");
```

Comment:
Uncaught ReferenceError: getElementById is not defined.
I don't understand why.